app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def users; end
+end
```

Comment:
- [ ] Rails do have some standard conventions for representing controller actions like `index` for main  `show` for details etc. to point to specific routes. In that case, it will be nice to have `index` and `show` for now. Leaving the action empty for now is cool.

See example:
```ruby
class UsersController < ApplicationController
  
  def index; end

  def show; end

end
```

Reply:
Thanks for the hint!